newIDE/app/src/EventsFunctionsList/index.js

```diff
@@ -86,6 +89,11 @@ export default class EventsFunctionsList extends React.Component<Props, State> {
     searchText: '',
   };
 
+  _togglePrivate = eventsFunction => {
+    eventsFunction.togglePrivate();
+    this.forceUpdate();
```

Comment:
Not sure if this is necessary or the way to go

Reply:
You're changing the value of something that is stored in libGD.js, in a memory that is not understood by React as a "state". So you need to call this.forceUpdate to tell React "hey I know I've not modified a state or a prop, but something still changed and you need to render again this component".

Reply:
Ok, I was unsure because some component seem to ask the component above via a prop to forceUpdate or something like this.